src/index.js

```diff
@@ -8,8 +9,23 @@ const defaultOptions = {
   source: true,
 };
 
+const defaultMtrcConf = {
+  h1: H1,
+  h2: H2,
+  h3: H3,
+  h4: H4,
+  h5: H5,
+  h6: H6,
+  code: Code,
+  p: P,
+  a: A,
+  li: LI,
+  ul: UL,
+};
+
 export default {
-  addWithInfo(storyName, info, storyFn, _options) {
+  addWithInfo(storyName, info, storyFn, _options, _mtrcConf) {
```

Comment:
I check it on my example (with and without info parameter). It works without errors so I change it as you wish.
But, if someone miss info parameter there is no sense to add mtrcConf object because it's redundant. MTRC configuration passed by mtrcConf change style only in description (correct?) so if there is no info what is a point to add mtrcConf?

MTRC is used only here:
```js
<div style={stylesheet.infoContent}>
  {MTRC(source).tree}
</div>
```
That is the reason I put it into separate prop.